api_tests/base/test_serializers.py

```diff
@@ -52,6 +53,27 @@ def null_field(*args, **kwargs):
     def valued_field(*args, **kwargs):
         return 'http://foo.com'
 
+
+class TestNodeSerializerandRegistrationSerializer(ApiTestCase):
+    """
+    All fields on the Node Serializer other than the few we can serialize for withdrawals must be redeclared on the
+    Registration Serializer and wrapped in HideIfWithdrawal
+    """
+    def test_registration_serializer(self):
+
+        # fields that are visible for withdrawals
+        visible_on_withdrawals = ['contributors', 'date_created', 'description', 'id', 'links', 'registration', 'title', 'type']
+        # fields that do not appear on registrations
+        non_registration_fields = ['registrations']
```

Comment:
I will add some tests for the HideIfRegistration field.  I assumed I had added them when I added the field, but it looks like my original tests were just centered around retractions hmph.